foreman_maintain.gemspec

```diff
@@ -20,14 +20,15 @@ the Foreman/Satellite up and running."
   s.require_paths = ['lib']
   s.executables = ['foreman-maintain',
                    'foreman-maintain-complete', 'foreman-maintain-rotate-tar']
+  s.required_ruby_version = ">= 2.5", "< 4"
```

Comment:
a tad unrelated, but I wonder if we should bump this to 2.7 as there is no platform that we'd deploy it on that has anything older than 2.7 (see https://community.theforeman.org/t/deprecation-of-ruby-2-5/28817)

not that I know of any benefits for the code between 2.5 and 2.7, but putting an arbitrary lower bound on a release we don't use seems odd?

(this would also require changes in https://github.com/theforeman/foreman_maintain/blob/82bdd577dbe017c379d3aa80ce1d285e66f63890/.github/workflows/ruby_tests.yml#L10 but that's a different story)